helm/circles-infra-suite/values-production.yaml

```diff
@@ -63,6 +64,8 @@ contracts:
   proxyFactoryAddress: 0xa6B71E26C5e0845f74c812102Ca7114b6a896AB2
   safeAddress: 0x3E5c63644E683549055b9Be8653de26E0B4CD36E
   fallBackAddress: 0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4
+  multiSendAddress: 0xA238CBeb142c10Ef7Ad8442C6D1f9E89e07e7761
```

Comment:
Are they the same addresses than in chiado? Where did you get the addresses from?

Reply:
I've checked here: all good
https://github.com/safe-global/safe-deployments/tree/main/src/assets/v1.3.0